src/components/TakeAction.vue

```diff
@@ -113,9 +167,36 @@ export default {
     },
     methods: {
         nextPage () {
+            const previousPanel = this.panel
+            const nextPanel = this.panel + 1
+
+            //update previous panel's status
+            this.panelStatus[previousPanel] = 'completed'
+            //update next panel's status
+            this.panelStatus[nextPanel] = 'inProgress'
+            //move to next panel
             this.panel = this.panel += 1
```

Comment:
Not part of your change, but we should change this so it's less confusing anyways :)

```suggestion
            this.panel += 1
```